smac/epm/gaussian_process_mcmc.py

```diff
@@ -215,6 +218,11 @@ def _train(self, X: np.ndarray, y: np.ndarray, do_optimize: bool=True):
         self.models = []
         for sample in self.hypers:
 
+            if (sample < -50).any():
```

Comment:
Where do these numbers come from? 
Also, this should work without the if statements

Reply:
> Where do these numbers come from?
RoBO used 20 here, but I found that the code also works with 50, so I thought I generalize this a bit.
> Also, this should work without the if statements
Yes, but I found it to be faster like this.